packages/auth/amplify_auth_cognito/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 1.4.0
+
+### Features
+- feat(auth): TOTP MFA
+- feat(authenticator): TOTP MFA ([#3609](https://github.com/aws-amplify/amplify-flutter/pull/3609))
```

Comment:
not sure if feat(authenticator) should be here, the pr does not have any change in the auth package

Reply:
It did have a small change, but yeah I could have removed this.